projects/plugins/jetpack/extensions/plugins/publicize/components/connection/index.js

```diff
@@ -61,17 +61,20 @@ class PublicizeConnection extends Component {
 	}
 
 	render() {
-		const { disabled, enabled, id, label, name } = this.props;
+		const { disabled, enabled, id, label, name, profilePicture } = this.props;
 		const fieldId = 'connection-' + name + '-' + id;
 		// Genericon names are dash separated
 		const serviceName = name.replace( '_', '-' );
 
 		let toggle = (
-			<FormToggle
+			<ConnectionToggle
```

Comment:
I know this `let` wasn't introduced by you, but I wonder whether we can improve this part too on this PR:

* do not use `let` to store the component instance
* do not use the `<Disable />` component below
* pass the `disable` property to the toggle component
* pass the disable prop to the `<FormToggle />`

Anyway, we can deal with this in a follow-up task.